lib/okta/configuration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+
+module Okta
+  class Configuration < Common::Client::Configuration::REST
+    def base_path
+      'https://deptva-eval.okta.com/api/v1/'
```

Comment:
This should be pulled from the Settings object, not hard-coded.

Reply:
Understood